core/pom.xml

```diff
@@ -146,11 +146,6 @@
       <artifactId>jetty-http</artifactId>
       <scope>compile</scope>
     </dependency>
-    <dependency>
-      <groupId>org.eclipse.jetty</groupId>
-      <artifactId>jetty-continuation</artifactId>
```

Comment:
IIRC, there are some other configurations related to `jetty-continuation` that need to be cleaned up in `core/pom.xml`.

Reply:
I have updated the POM to reflect the absent of this package, I also I did another global search to make sure nothing got impacted.